torchbench.py

```diff
@@ -954,6 +1036,16 @@ def main():
         optimize_ctx = torchdynamo.optimize(dummy_fx_compile, nopython=args.nopython)
         experiment = speedup_experiment
         output_filename = "overheads.csv"
+    elif args.cold_start:
+        optimize_ctx = torchdynamo.optimize(
+            aot_autograd_speedup_strategy, nopython=args.nopython
+        )
+        experiment = cold_start_experiment
+        backend_str = "nvfuser" if args.nvfuser else "nnc"
+        output_filename = "cold_start_{backend_str}.csv"
+        args.isolate = True
+        # horace said so
```

Comment:
oops.. haha i should remove this. 

but i did want to confirm whether putting it just in this cold-start benchmark makes sense- isn't it something that should be applied across all the (maybe training only) benchmarks more uniformly?